titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/JooqUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.ext.jooqflyway.jobactivity;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Supplier;
+
+import org.jooq.DSLContext;
+import reactor.core.publisher.Mono;
+
+public class JooqUtils {
```

Comment:
Did we move or duplicate this class? Seems like it exists in `com.netflix.titus.ext.jooq` already.

Reply:
yes there is a lot of duplication, this should eventually replace the jooq folder